frontend/e2e/pages/patients.js

```diff
@@ -9,7 +9,10 @@ function addPatient(dobFormat) {
   const fullName = `${lastName}, ${firstName}`;
   const dob = dayjs(faker.date.between("1920-01-01", "2002-12-31"));
   const dobForInput = dob.format(dobFormat);
-  const dobForPatientLink = dob.format("MM/DD/YYYY");
+  // js months are 0-offset
+  const monthForPatientLink = dob.month() + 1;
```

Comment:
why Javascript??? just why?????

Reply:
Right!! Also fun fact, dob.day() returns the day of the week, 0-6, while dob.date() returns the day of the month 🙃